src/test/java/useCases/impactor/MoneyImpactorTest.java

```diff
@@ -30,7 +35,7 @@ static void setup() {
      */
     @Test
     public void testDeductNormal() {
-        for (int i = -1000; i < 1000; i++) {
+        for (int i = LOWER_LIMIT; i < UPPER_LIMIT; i++) {
             MoneyImpactor.deduct(i, giver);
             assertEquals(1000-i, giver.getMoney());
             giver.setMoney(1000);
```

Comment:
Hmm, maybe you want a local variable just called, "defaultMoney" or something here?